package-lock.json

```diff
@@ -1,8 +1,19121 @@
 {
   "name": "ngx-skeleton-loader-demo",
   "version": "2.10.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
These changes are not necessary. Can you revert that, please?

Reply:
Sure I'll revert them. When I do a `npm i` and the `package-lock.json` changes I'm accustomed to push it, because it is out of date.

Also I will look at the pipeline.

Reply:
Hi @rkristelijn Actually, this `package-lock.json` is up-to-date, so I can say the changes are related to some local configuration you have in your local setup.

Regards to the bundlesize update, the approach is correct. This is not related to budget, which is integrated on Angular CLI, by default. This command is checking the **final bundle for the minified and uglified** bundled file, so the changes are expected and there's no need to change these steps based on their intention, which is to create awareness about bundle size concerns on the final bundle.

Also, this issue is because you're not using the predefined NodeJS version as addressed in `.nvmrc` file (https://github.com/willmendesneto/ngx-skeleton-loader/blob/master/.nvmrc)

Thanks for raising the issue, btw. We just found the root cause, so I'm closing the issue since it's related to local setup and using a different NodeJS version, not affecting the final bundle nor other tasks.